Testing/VelaptorTests/Content/TextureTests.cs

```diff
@@ -291,14 +293,16 @@ public void InternalCtor_WhenInvoked_UploadsTextureDataToGpu()
             0,
             GLPixelFormat.Rgba,
             GLPixelType.UnsignedByte,
-            expectedPixelArray), Times.Once());
+            Arg.Is<byte[]>(actualPixelArray => actualPixelArray.SequenceEqual(expectedPixelArray)));
```

Comment:
@KinsonDigitalAdmin I'm not sure about this refactor. More info here: https://stackoverflow.com/questions/12699066/nsubstitute-checking-received-methods-with-array-arguments

Let me know if you have a better solution

Reply:
This works fine.  NSubstitute basically sends the arg to the lambda method, which is, in your case, `actualPixelArray.`  There is another way by using `Callback` but this is more elegant is is how you should do this.